rust/src/named_graph.rs

```diff
@@ -23,6 +23,14 @@ predicates! {
         }
     }
 
+    pub semidet fn delete_named_graph(context, store_term, graph_name_term, found_term) {
```

Comment:
instead of returning found as a boolean, it is more prology to make the predicate fail if the database could not be found.

you can easily do this with
```
into_prolog_result(found)
```
which will return a PrologResult that is a failure if found is false, and a success when found is true.

Reply:
I thought you didn't want `delete_named_graph` to fail if the named graph doesn't exist. That's for a couple of reasons:
* I believe that the “remove triple” operation (or another operation?) doesn't fail if the triple doesn't exist.
* In [`delete_label` in `directory.rs`](<https://github.com/terminusdb/terminusdb-store/blob/07e4662ef544fb65eef6b66a280f5798fee9fdb6/src/storage/directory.rs#L374-L394>), the error with kind `io::ErrorKind::NotFound` is returned as `Ok(false)`. If the not finding the file is a failure, shouldn't it return the error?